regression/memory-analyzer/pointer_to_struct_01/test.desc

```diff
@@ -1,8 +1,7 @@
 CORE
 main.exe
 --breakpoint checkpoint --symbols p
-struct S tmp;
-tmp = \{ \.next=\(\(struct S \*\)0\) \};
-p = \&tmp;
+st = \{ .next=\&st \};
+p = \&st;
```

Comment:
Could you please squash this into the commit that made this possible?

Reply:
It was this one: https://github.com/diffblue/cbmc/pull/4578/commits/33d0a79b6843c2c6c1b60ee8c8540dfe9df9da86. I will squash them together.